From 58867bd1c28be47d22526a152353e3ec812e6321 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Tue, 16 Sep 2014 20:20:10 -0400
Subject: [fs] nfsd4: fix out of date comment

Message-id: <1410898849-22126-127-git-send-email-bfields@redhat.com>
Patchwork-id: 95733
O-Subject: [Patch RHEL7 126/165] nfsd4: fix out of date comment
Bugzilla: 1044798 1123721 1141932
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 83e452fee81cf67a8e08fd843291a7cff62a3dc7
Bugzilla: 1044798
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/state.h b/fs/nfsd/state.h
index 73a209d..0b23450 100644
--- a/fs/nfsd/state.h
+++ b/fs/nfsd/state.h
@@ -345,7 +345,7 @@ struct nfs4_stateowner {
  struct list_head   so_stateids;
  struct nfs4_client   *so_client;
  const struct nfs4_stateowner_operations *so_ops;
- /* after increment in ENCODE_SEQID_OP_TAIL, represents the next
+ /* after increment in nfsd4_bump_seqid, represents the next
   * sequence id expected from the client: */
  atomic_t    so_count;
  u32     so_seqid;
-- 
1.7.1